docs/OnTrack/Voice Verification/Handbook for Voice Verification.md

```diff
@@ -0,0 +1,85 @@
+# Voice Verification Handbook
```

Comment:
You can use the [OnTrack Documentation Template](https://github.com/thoth-tech/documentation/blob/main/docs/OnTrack/Documentation/OnTrack%20Documentation%20Template.md), which is a good way of outlining the version of the document, who worked on it, and setting up the remainder of the document.

Reply:
Additionally, considering that this is a handbook, this document should contain links to other documentation produced for the Voice Verification team. This would include design docs, SRS document, and any other documentation produced to date. Guiding new developers through what the documents are, and where to find them is super important to ensure that we aren't dropping newcomers in the deep end.